src/Workspaces/Remote/Core/Storage/RemotePersistentStorageLocationService.cs

```diff
@@ -44,7 +44,8 @@ public static void UpdateStorageLocation(SolutionId id, string storageLocation)
                 }
                 else
                 {
-                    _idToStorageLocation[id] = storageLocation;
+                    // Store the esent database in a different location for the out of proc server.
+                    _idToStorageLocation[id] = Path.Combine(storageLocation, "Server");
```

Comment:
this looks like the root cause of why suddenly RPS test says sqlite is loaded in VS for perf tests.

basically, OOP creates sqlite file and VS pick that up and think solution already have persisted storage enabled and try to use it.